src/Serilog.Sinks.NLog/project.json

```diff
@@ -0,0 +1,27 @@
+﻿{
+  "version": "2.0.0-*",
```

Comment:
There's already a 2.x of this sink published - can we update this to 3.0.0-* please?